ipywidgets/static/widgets/js/widget_state.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// npm compatibility
+if (typeof define !== 'function') { var define = require('./requirejs-shim')(module); }
+
+define(["base/js/namespace"], function(Jupyter) {
+    "use strict";
+
+    var save_state = function() {
```

Comment:
Like @minrk suggests, you should probably implement this as an action.  Don't add it to the toolbar, and if you want, you don't even have to work out how to add it to the menu.  If implemented as an action, you'll be able to access it through the command palette.